macros/fitScanData.py

```diff
@@ -1,87 +1,103 @@
+class DeadChannelFinder(object):
+    def __init__(self):
+        import numpy as np
```

Comment:
Is this the correct way to do this?  Looks like multiple classes/methods inside `DeadChannelFinder` and `ScanDataFitter` require numpy.